ops/terraform/services/base/values/prod.yaml

```
@@ -27,30 +27,33 @@
 /bfd/prod/migrator/nonsensitive/db_migrator_tmp_dir: /opt/bfd-db-migrator/tmp
 
 # PIPELINE CONFIGURATION
-/bfd/${env}/pipeline/nonsensitive/bfd_pipeline_dir: &bfdPipelineDir /bluebutton-data-pipeline
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_dir: *bfdPipelineDir
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_new_relic_metric_host: *newRelicMetricHost
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_new_relic_metric_path: *newRelicMetricPath
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_user: bb-etl
-/bfd/${env}/pipeline/nonsensitive/instance_type: m5.4xlarge
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_micrometer_cw_enabled: true
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_micrometer_cw_namespace: bfd-${env}/bfd-pipeline
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_micrometer_cw_interval: PT1M
+/bfd/${env}/pipeline/shared/nonsensitive/bfd_pipeline_dir: &bfdPipelineDir /bluebutton-data-pipeline
+/bfd/${env}/pipeline/shared/nonsensitive/data_pipeline_dir: *bfdPipelineDir
+/bfd/${env}/pipeline/shared/nonsensitive/data_pipeline_new_relic_metric_host: *newRelicMetricHost
+/bfd/${env}/pipeline/shared/nonsensitive/data_pipeline_new_relic_metric_path: *newRelicMetricPath
+/bfd/${env}/pipeline/shared/nonsensitive/data_pipeline_user: bb-etl
+/bfd/${env}/pipeline/shared/nonsensitive/data_pipeline_micrometer_cw_enabled: true
+/bfd/${env}/pipeline/shared/nonsensitive/data_pipeline_micrometer_cw_namespace: bfd-${env}/bfd-pipeline
+/bfd/${env}/pipeline/shared/nonsensitive/data_pipeline_micrometer_cw_interval: PT1M
 ## PIPELINE CCW
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_ccw_rif_job_enabled: false
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_filtering_non_null_and_non_2023_benes: true
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_idempotency_required: false
+/bfd/${env}/pipeline/ccw/nonsensitive/data_pipeline_rda_job_enabled: false # RDA Jobs are disabled by default on CCW Pipelines
+/bfd/${env}/pipeline/ccw/nonsensitive/data_pipeline_ccw_rif_job_enabled: false
+/bfd/${env}/pipeline/ccw/nonsensitive/data_pipeline_filtering_non_null_and_non_2023_benes: true
+/bfd/${env}/pipeline/ccw/nonsensitive/data_pipeline_idempotency_required: false
+/bfd/${env}/pipeline/ccw/nonsensitive/instance_type: m6i.4xlarge
```

Let's shrink this a bit, especially since we've historically gotten away with 25% of the system memory and we're bumping that up to 80%.
```suggestion
/bfd/${env}/pipeline/ccw/nonsensitive/instance_type: c6i.4xlarge
```